bundles/org.openhab.binding.tado/src/main/java/org/openhab/binding/tado/internal/handler/TadoZoneHandler.java

```diff
@@ -90,7 +93,9 @@ public OverlayTerminationCondition getDefaultTerminationCondition() throws IOExc
     }
 
     public ZoneState getZoneState() throws IOException, ApiException {
-        return getApi().showZoneState(getHomeId(), getZoneId());
+        // null pointer exception fixed by Andrew Fiddian-Green
```

Comment:
No need to put in specific comments like these.  You can make a remark in the commit message. If this has a issue number or other references I would mention that in the git commit message.

Reply:
Ok. Done.